app/src/main/java/org/simple/clinic/widgets/qrcodescanner/QrCodeScannerView.kt

```diff
@@ -19,7 +22,10 @@ constructor(
     defStyleAttr: Int = 0
 ) : FrameLayout(context, attrs, defStyleAttr) {
 
+  val scans = PublishSubject.create<String>()
```

Comment:
Can we make this a private variable and expose it to callers as an `Observable` separately?